test/checkOutputFile.m

```diff
@@ -33,24 +33,15 @@ function checkOutputFile(testCase,scenarioFolder,scenarioFileName,...
 % limitations under the License.
 
 % Check if the same file was passed
-% assertEqual(testCase,scenarioFileName,exampleFileName)
+assertEqual(testCase,scenarioFileName,exampleFileName)
 
 scenarioFilePath = fullfile(scenarioFolder, scenarioFileName);
 exampleFilePath = fullfile(exampleFolder, exampleFileName);
 
-if isNodesPosition(scenarioFilePath)
-    scenarioOut = readNodesPosition(scenarioFilePath);
-    exampleOut = readNodesPosition(exampleFilePath);
-elseif isQdFile(scenarioFilePath)
+if isQdFile(scenarioFilePath)
     sortBy = {'delay','pathGain','phaseOffset','aodEl','aodAz','aoaEl','aoaAz'};
     scenarioOut = readQdFile(scenarioFilePath, sortBy);
     exampleOut = readQdFile(exampleFilePath, sortBy);
-elseif isMpcCoordinates(scenarioFilePath)
-    scenarioOut = readMpcCoordinates(scenarioFilePath);
-    exampleOut = readMpcCoordinates(exampleFilePath);
-elseif isNodePositions(scenarioFilePath)
-    scenarioOut = readNodePositions(scenarioFilePath);
-    exampleOut = readNodePositions(exampleFilePath);
```

Comment:
These are the old files for the visualizer, which we do not support anymore

Reply:
i don't remember about this, are we really not supporting the MpcCoordinates anymore?
Is this about the json output change?
If so, that should be checked for equality, instead.
I think it's still important to check ALL the (possible) output files, or bugs may be hidden behind this lack of testing

Reply:
Yes that's for the json and I have introduced the functions isMpcJsonCoordinates and isNodePositionsJson. I haven't just replaced them because for a period we have supported both.